### ISSUE TITLE
HELP:  Disable annoying notifications

### ISSUE THREAD

**Issue by MagicalASO** [LazyVim/LazyVim#2300]
### Did you check the docs?

- [X] I have read all the LazyVim docs

### Is your feature request related to a problem? Please describe.

these popups are annoying and stays on screen for a while, every time i open neovim.
I tried fiddling with docs but still could not manage to make those popups go away. A one-time i wanted to punch through the screen but somehow held myself back. 
![hjvbvkj](https://github.com/LazyVim/LazyVim/assets/150692312/c1f6082a-e211-4e00-b0e7-1198b4baef82)

Also earlier Ctrl+c,Ctrl+z ( a lot quicker than :qa! )would let me escape out of neovim without saving any changes but after installing Lazyvim I can exit neovim but messy changes gets saved.

### Describe the solution you'd like

Disable notification popups.
remap ctrl+c,ctrl+z to exit without saving anything.

### Describe alternatives you've considered

.

**Comment by dpetka2001** [comment]
For notifications see #1927. Not sure about saving.

**Comment by dpetka2001** [comment]
About the saving you need to set `vim.opt.autowrite = false` in your `/lua/config/options.lua`. That should solve the issue with `<ctrl-z>`.

**Comment by abeldekat** [comment]
Hello @MagicalASO 

I am wondering: Why did you not ask your questions in the [discussions](https://github.com/LazyVim/LazyVim/discussions) section?

A feature request should not start with "HELP". Before *creating* a bug report of pull request, please also read the [lazy.nvim](https://github.com/folke/lazy.nvim#%EF%B8%8F-configuration) documentation.

Quoting:
> LazyVim is a Neovim setup powered by [💤 lazy.nvim](https://github.com/folke/lazy.nvim) to make it easy to customize and extend your config.

Kindly close this issue.

**Comment by MagicalASO** [comment]
> For notifications see #1927. Not sure about saving.
thanks! the autowrite got fixed but those notification still not gone.
![fff](https://github.com/LazyVim/LazyVim/assets/150692312/06a8185e-b077-4dc3-a581-4e63c1dba7c8)

I also set rcarriga/nvim-notify to false in disabled.lua at .config/nvim/lua/plugins but no luck.
everytime whatever changes i made about this notify plugin, the notification looked like this!
![gvkg](https://github.com/LazyVim/LazyVim/assets/150692312/a731cb4b-a8fd-4519-9e80-9d32b94833af)

now its look less annoying. I was also using lazyvim on termux where this notification blocks  almost entire screen and i have to wait few seconds.
it should not be showing same notification every time i open the neovim ? 
A few time i get the notification but i think no one wants same notification every single time.
how can i set `timeout` value of notify plugin to 0 ?
